gimie/sources/github.py

```diff
@@ -284,6 +285,11 @@ def _get_user(self, node: Dict[str, Any]) -> Person:
             affiliations=orgs,
         )
 
+    def _get_license(self):
+        """Extract a SPDX License URL from a GitHub Repository"""
+        self.license = get_license(repo_url=self.url, headers=self._set_auth())
```

Comment:
In the future, `get_license` should take a list of `File` objects. This would allow it to work with Git or Github.